libsel4utils/include/sel4utils/benchmark_track.h

```diff
@@ -62,14 +62,14 @@ static inline void seL4_BenchmarkTrackDumpFullSyscallLog(benchmark_track_kernel_
 
     while (logBuffer[index].start_time != 0 && (index * sizeof(benchmark_track_kernel_entry_t)) < logSize) {
         if (logBuffer[index].entry.path == Entry_Syscall) {
-            fprintf(fd, "|     %-15d|     %-15d|     %-15llu|     %-15d|     %-15d|     %-15d|     %-15d|\n",
-                    index,
-                    logBuffer[index].entry.syscall_no,
-                    (uint64_t) logBuffer[index].start_time,
-                    logBuffer[index].duration,
-                    logBuffer[index].entry.cap_type,
-                    logBuffer[index].entry.invocation_tag,
-                    logBuffer[index].entry.is_fastpath);
+            fprintf(fd, "|     %-15lu|     %-15lu|     %-15"PRIu64"|     %-15lu|     %-15lu|     %-15lu|     %-15lu|\n",
```

Comment:
There is "*" in the printf() format args that we could use here to make this generic:
```suggestion
           #define WORD_INDENT ((CONFIG_WORD_SIZE / 4) - 1)
            fprintf(fd, "|     %-*"PRIu64"|     %-*"PRIu64"|     %-*"PRIu64"|     %-*"PRIu64"|     %-*"PRIu64"|     %-*"PRIu64"|     %-*lu|\n",
                    WORD_INDENT, index,
                    WORD_INDENT, logBuffer[index].entry.syscall_no,
                    WORD_INDENT, logBuffer[index].start_time,
                    WORD_INDENT, logBuffer[index].duration,
                    WORD_INDENT, logBuffer[index].entry.cap_type,
                    WORD_INDENT, logBuffer[index].entry.invocation_tag,
                    WORD_INDENT, logBuffer[index].entry.is_fastpath);            
```